ibis/backends/oracle/datatypes.py

```diff
@@ -32,6 +32,8 @@ def to_ibis(cls, typ, nullable=True):
                     scale=typ.scale or 0,
                     nullable=nullable,
                 )
+        elif isinstance(typ, sat.String):
+            return dt.String()
```

Comment:
Is this not covered by the base class implementation?

Reply:
I thought it would be, but apparently not?  I'll take a look at the base implementation to see what's going sideways.

Reply:
Ahh, Jim noticed we aren't doing a subclass check, so `if dtype := _from_sqlalchemy_types.get(type(typ))` isn't catching things like the oracle `LONG` and `NVARCHAR2`.

I can move the `isinstance` check into the base alchemy file so we have it as a fallback for all the backends.